src/api/status/public/css/terminal.css

```diff
@@ -1,4 +1,23 @@
 .xterm {
   padding: 0.5rem;
   height: 89vh;
+  /* padding-right: 8rem;
```

Comment:
Let's not do this.  If you don't need it, remove it.

Reply:
Oops, how did I miss that.